automated-testing/unit-testing/custom-connector.md

```diff
@@ -0,0 +1,11 @@
+# Custom Connector Testing
+
+When developing Custom Connectors to put data into the Power Platform there are some strategies you can follow:
+
+## Unit Testing
+
+There are several verifications one can do while developing custom connectors in order to be sure the code itself is working properly. There are two main ones: validating the OpenAPI schema which the connector is defined and validating if the schema also have all the information necessary for the certified connector process(the later one is optional, but necessary in case you want to publish it as a certified connector).
+
+For testing if OpenAPI schema is valid, there are several tool available. A list of them are available in this [link](https://openapi.tools/#description-validators). CSE Business Apps teams have been using [swagger-cli](https://github.com/APIDevTools/swagger-cli).
```

Comment:
The content we are creating here is open to everyone, not all might be familiar with CSE. Can you reword the last sentence to come across as a suggestion?